integration/app_integration_test.go

```diff
@@ -1397,6 +1437,73 @@ func (p *pack) waitForLogout(appCookie string) (int, error) {
 	}
 }
 
+func (p *pack) startRootAppServers(t *testing.T, count int, extraApps []service.App) []*service.TeleportProcess {
+	log := testlog.FailureOnly(t)
+
+	var err error
+	servers := make([]*service.TeleportProcess, count)
+
+	for i := 0; i < count; i++ {
+		raConf := service.MakeDefaultConfig()
+		raConf.Console = nil
+		raConf.Log = log
+		raConf.DataDir, err = ioutil.TempDir("", fmt.Sprintf("app-server-%d-%s", i, p.rootCluster.Secrets.SiteName))
```

Comment:
nit: `ioutil` is deprecated, use os.MkdirTemp:
```suggestion
		raConf.DataDir, err = os.MkdirTemp("", fmt.Sprintf("app-server-%d-%s", i, p.rootCluster.Secrets.SiteName))
```